includes/Documents/OrderDocument.php

```diff
@@ -1477,6 +1478,59 @@ public function get_pdf() {
 		return apply_filters( 'wpo_wcpdf_get_pdf', $pdf, $this );
 	}
 
+	/**
+	 * Replace .avif images with .jpg images in HTML
+	 * @param string $html The HTML to process
+	 * 
+	 * @return string The processed HTML
+	 */
+	public function replace_avif_images( string $html ): string {
```

Comment:
This should be placed in the `functions.php` file so it could be reused elsewhere if needed.